### ISSUE TITLE
The current row is not updated till we save

### ISSUE THREAD

**Issue by afroz786** [vitalets/angular-xeditable#470]
e-ng-change($data,currentRow) in this on change event i want the value of different fields in current row. m getting the current data but since the currentRow is not bind at the moment i get undefined. but i want to do validation of 1 select field depending on other select field. How to do that?

Thanks for speedy reply, u already solved 2 of my issues :+1:

**Comment by ckosloski** [comment]
I think the only thing you can currently do is some form of onbeforesave/onaftersave.  See the [documentation](http://vitalets.github.io/angular-xeditable/#editable-form)

**Comment by afroz786** [comment]
Then after every row the user has to save according to my current requirement. Guess have to make use of normal select box and table for this.

**Comment by ckosloski** [comment]
Yeah, I don't think there is any other way to do it right now and I think I've seen other issues opened with similar problems.
